code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -149,8 +149,8 @@ update_flag
 (note: colors has to be applied every icon update)
 */
 
-	if(destroyed)
```

Comment:
(all?) the `destroyed` var usages are removed, but the var itself still exists. Should have cleaned that up as well, ideally